src/_guides/language/effective-dart/usage.md

```diff
@@ -119,6 +119,60 @@ library. Follow these two rules:
 * An import path should never contain `/lib/`.
 * A library under `lib` should never use `../` to escape the `lib` directory.
 
+## Booleans
+
+### DO use `??` to convert `null` to a Boolean value.
+
+You sometimes have an expression that can evaluate to `true`, `false`, or `null`
+and you need to pass the result to something that doesn't accept `null`. A
+common case is the result of a null-aware method call being used as a condition:
+
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (null-aware-condition)"?>
+{% prettify dart %}
+if (optionalThing?.isEnabled) {
+  print("Have enabled thing.");
+}
+{% endprettify %}
+
+This code will throw an exception if `optionalThing` is `null`. To fix this, you
+need to "convert" the `null` value either to `true` or `false`. There are two
+ways to do this, using `??` or `==`:
+
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (convert-null-aware)"?>
+{% prettify dart %}
+// If you want null to be false:
+optionalThing?.isEnabled ?? false;
+
+// If you want null to be true:
+optionalThing?.isEnabled ?? true;
+{% endprettify %}
+
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (convert-null-equals)"?>
+{% prettify dart %}
+// If you want null to be false:
+optionalThing?.isEnabled == true;
+
+// If you want null to be true:
+optionalThing?.isEnabled == false;
+{% endprettify %}
+
+Both operations produce the same result and do the right thing, but `??` is
+preferred for three main reasons:
+
+*   The `??` operator clearly signals that the code has something to do with a
+    `null` value.
+
+*   The `== true` looks like a common new programmer mistake where the equality
+    operator is redundant and can be removed. That's true when the Boolean
```

Comment:
GLOBAL: Boolean -> boolean (except when it's the first word in a section head, of course)